misk/src/main/kotlin/misk/web/actions/NotFoundAction.kt

```diff
@@ -38,4 +50,40 @@ class NotFoundAction @Inject constructor() : WebAction {
       )
     }
   }
+
+  /** Generates a message for a client that hints what may have been wrong with their request. */
+  private fun notFoundMessage(path: String, httpCall: HttpCall) = buildString {
+    val boundActions = servletProvider.get().boundActions
+    val alternativeActions = boundActions.mapNotNull {
+      it.matchByUrl(httpCall.url)
+    }.filterNot { it.action.pathPattern.matchesWildcardPath }
+
+    append("Nothing found at /$path.\n\n")
+    append("Received:\n")
+    append("  ${httpCall.dispatchMechanism.method} /$path\n")
+    for (accept in httpCall.accepts()) {
+      append("  Accept: $accept\n")
```

Comment:
nit: have the `Accept` line follow the `Content-Type` as in the alternative (or the other way around)

Reply:
Fixed